src/test/java/com/revature/cmsforce/CMSforceApplicationTest.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.cmsforce;
+
+import org.testng.annotations.Test;
+
+import com.revature.cmsforce.CMSforceApplication;
+
+public class CMSforceApplicationTest {
+
+	// Test class added ONLY to cover main() invocation not covered by application tests.
+	@Test
+	public void mainTest() {
```

Comment:
Is this a meaningful test? Test coverage is not meaningful if it's not actually testing anything

Reply:
I contemplated that repeatedly, it was a choice that I made. It is not a very meaningful test hence the comment above the method itself. I was researching the importance of full coverage and when would it be b=necessary to test the main method. It seems it is only for aesthetic purposes and differs from company to company. I can remove it if you would like.,

Reply:
Gotcha. I understand the motivation but I would remove it